css/main.css

```diff
@@ -307,21 +307,132 @@ li {
 }
 
 .list-title {
-  font-family: "Inter", sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
 }
 
 .languages {
-  list-style: inside url("../media/svg/rectangle.svg");
+  list-style: inside url('../media/svg/rectangle.svg');
 }
 
 .frameworks {
-  list-style: inside url("../media/svg/square.svg");
+  list-style: inside url('../media/svg/square.svg');
 }
 
 .skills {
-  list-style: inside url("../media/svg/circle.svg");
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+/* button styles */
+.btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  background: #fff;
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+  padding: 8px 10px;
+  width: fit-content;
+  transition: all 0.3s ease-in-out 0s;
+  display: flex;
+  align-items: center;
+}
+
+.btn:hover {
+  background: #6070ff;
+  color: #fff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  cursor: pointer;
+}
+
+.btn:disabled {
+  color: #5e6c84;
+  border: 1px solid #5e6c84;
+}
+
+/* Contact Form */
+.footer {
+  background: url('../images/background/header-shapes-contact@2x.svg');
+  background-color: #6070ff;
+  border-top-left-radius: 90px;
+}
+
+.footer .contact {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 100px 0;
+}
+
+.footer .contact h2 {
+  margin-top: 0;
+}
+
+.footer .contact h2,
+.footer .contact p {
+  text-align: initial;
+  color: #ebebff;
+}
+
+.footer .contact p {
+  font-size: 40px;
+  line-height: 52px;
+  transition: transform 2s ease-in-out 0s;
+}
```

Comment:
Kindly change the color of your `<p>` to black and change the font-weight according to the Figma design  like that :

:heavy_check_mark: 
`font-family: Inter;
font-style: normal;
font-weight: bold;
color:black
`